ecommerce/frontend/src/components/UserReview.js

```diff
@@ -0,0 +1,128 @@
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                    {
+                        comment:
+                            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Jeff Einstein',
+                        date: 'December 23',
+                        rating: 4,
+                    },
+                    {
+                        comment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.',
+                        author: 'Hasan Kaya',
+                        date: 'December 11',
+                        rating: 3,
+                    },
+                ]
+                const urlStr = `/product/${productId}/review?pagesize=${pageSize}&currentpage=${currentPage}`
+                console.log(urlStr)
+                // const { data: reviews } = await api.get(urlStr)
+                setReviews(tmp)
+                await sleep(2000)
+            } catch (error) {
+                console.error('Failed to load user reviews', error)
+            } finally {
+                setIsLoading(false)
+            }
+        }
+        fetch()
+    }, [currentPage])
+
+    const onPaginationChanged = value => {
+        // When page count is changed, current page is set
+        console.log('Page changed to ', value)
+        setCurrentPage(value)
+    }
+
+    const Review = ({ review: { comment, author, date, rating } }) => {
+        return (
+            <div className="review-item-content">
+                <div className="rate-and-message">
+                    <div className="rate">
+                        <Rate defaultValue={rating} disabled={true}></Rate>
+                    </div>
+                    <div className="message">
+                        <p>{comment}</p>
+                    </div>
+                </div>
+                <div className="author-and-date">
+                    <Link to="/">
+                        <div className="author">
+                            <p>{author}</p>
+                        </div>
+                    </Link>
+                    <div className="date">
+                        <p>{date}</p>
+                    </div>
+                </div>
+            </div>
```

Comment:
styles applied to classNames are document-scoped not component-scoped. this means that then class names could conflict with other components. please prefix each class-name with `user-review-` for example